### PR TITLE
Quarto drafts for 13 cheat sheets

### PR DESCRIPTION
**Pull request by averiperny**
Current drafts of quarto cheat sheets, I will follow up with specific questions for each one.

### CODE REVIEW

File: rmarkdown/rmarkdown-accessible.qmd [hunk 1]
```diff
@@ -0,0 +1,538 @@
...
+
+Output data frames as tables using `kable(data, caption)`.
+
+``` {{r}}
+data <- faithful[1:4,]
+knitr::kable(data,
+             caption = "Tables with kable")
+```
+
+Other table packages include **flexible**, **gt**, and **kableExtra**.
```
Comment (cwickham) [inline rmarkdown/rmarkdown-accessible.qmd:187]: I think "flexible" should be "flextable"

File: rmarkdown/rmarkdown-accessible.qmd [hunk 2]
```diff
@@ -0,0 +1,538 @@
...
+| HTML Tabsets                                    | ::: panel-tabset                                                         |
+|                                                 | ## Plots                                                                 |
+|     # Results                                   |                                                                          |
+|     ## Plots text                               | text                                                                     |
+|     text                                        |                                                                          |
+|                                                 | ## Tables                                                                |
+|     ## Tables                                   |                                                                          |
+|     more text                                   | more text                                                                |
+|                                                 | :::                                                                      |
```
Comment (cwickham) [inline rmarkdown/rmarkdown-accessible.qmd:295]: Markdown doesn't match the output here, e.g. missing `::: panel-tabset` in left hand column

**Review by mine-cetinkaya-rundel — COMMENTED** [review]
- [x] dplyr
- [x] forcats
- [x] keras
- [x] lubridate
- [x] devtools
- [x] plumber
- [x] purrr
- [x] reticulate
- [x] rmarkdown
- [x] shiny
- [ ] sparklyr
- [x] stringr
- [x] tidyr
- [x] data-import

**Comment by mine-cetinkaya-rundel** [comment]
Leaving sparklyr sheet as is for now since it's based off the older version of the cheatsheet.

**Pull request merged**